project-addons/custom_emails_company/models/invoice.py

```diff
@@ -0,0 +1,37 @@
+# Necesario para el envio del email de forma automatica justo despues de la validación
+from odoo import models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    def action_invoice_open(self):
+
+        res = super(AccountInvoice, self).action_invoice_open()
+        if not self.not_send_email and self.type == 'out_invoice':
```

Comment:
Esto hay que hacerlo dentro de un for, ya que pueden venir varias facturas al mismo tiempo en self y daría un error de singleton

Reply:
Hecho.